buildSrc/src/test/groovy/org/springframework/cloud/pipelines/ProjectCustomizerSpec.groovy

```diff
@@ -109,4 +111,8 @@ class ProjectCustomizerSpec extends Specification {
 			}
 	}
 
+	String filePath(String canonicalPath) {
+		canonicalPath.replaceAll("/", Matcher.quoteReplacement(File.separator))
```

Comment:
Can you add `return`? I personally always prefer it to be there in Groovy code

Reply:
Absolutely!